Gemfile.lock

```diff
@@ -32,7 +32,7 @@ GEM
     diff-lcs (1.3)
     docile (1.1.5)
     highline (1.7.8)
-    i18n (0.8.1)
+    i18n (0.8.4)
```

Comment:
Ah, this happened again: https://github.com/codeclimate/codeclimate/pull/658#issuecomment-301601663

just make sure to do a bit of QA locally before shipping to make sure this doesn't break anything in a way the tests aren't catching